### PR TITLE
HPCC-15187 DOCS:Options dropdown in QuerySetQueryWidget

### PR DESCRIPTION
**Pull request by g-pan**
FIX HPCC-15187 DOCS:Options dropdown in QuerySetQueryWidget
Update the ECL Watch manual the Queries section in particular
to show not only the Options action button, but to document
all the action buttons on the queries page/tab, as none were
previously documented.

Signed-off-by: G Panagiotatos greg.panagiotatos@lexisnexis.com
@JamesDeFabia please review
@miguelvazq please review

### CODE REVIEW

File: docs/ECLWatch/ECLWa_mods/ECLWatchQueries.xml [hunk 1]
```diff
@@ -158,6 +158,148 @@ depending on where they wind up.-->
...
+                    <para>Suspended queries.</para>
+                  </listitem>
+
+                  <listitem>
+                    <para>Active queries.</para>
+                  </listitem>
+                </itemizedlist>
+
+                <para>The Filter also also supports wild cards. To limit or
+                filter your search results you can use keywords as displayed
```
Comment (JamesDeFabia) [inline docs/ECLWatch/ECLWa_mods/ECLWatchQueries.xml]: Remove the second sentence, keywords do not apply.

**Comment by miguelvazq** [comment]
The screenshots look current.

**Comment by JamesDeFabia** [comment]
@g-pan Looks Good to merge now

**Comment by g-pan** [comment]
good to merge.

**Pull request merged**